src/SLCore/Common/Helpers/ConnectionIdHelper.cs

```diff
@@ -33,6 +35,22 @@ internal class ConnectionIdHelper : IConnectionIdHelper
         private const string SonarQubePrefix = "sq|";
         private static readonly Uri SonarCloudUri = new Uri("https://sonarcloud.io");
 
+        public string GetConnectionIdFromUri(Uri uri, string organisation)
+        {
+            if (uri is not null)
```

Comment:
invert if to reduce nesting

Reply:
it will not be that useful because then we'll need to have more return points (since this way it returns null when organisation is missing)